packages/bezier-react/src/components/Forms/SegmentedControl/SegmentedControl.tsx

```diff
@@ -40,26 +42,23 @@ function SegmentedControlItemListImpl<
   style: styleProp,
   className: classNameProp,
   ...rest
-}: SegmentedControlItemListProps<Type, Value>, forwardedRef: React.Ref<HTMLDivElement>) {
-  const [selectedElement, setSelectedElement] = useState<HTMLButtonElement | null>(null)
+}: SegmentedControlItemListProps<Type, Value>) {
+  const [index, setIndex] = useState<number | null>(null)
 
   const {
     type,
     size,
     width,
   } = useSegmentedControlContext('SegmentedControlItemList')
 
-  const {
-    containerRef: ref,
-    render: renderIndicator,
-  } = useSegmentedControlIndicator({
-    target: selectedElement,
-    refs: [forwardedRef],
-  })
-
-  const contextValue = useMemo(() => ({
-    setSelectedElement,
-  }), [])
+  const contextValue: SegmentedControlItemListContextValue = useMemo(() => ({
+    index,
+    length: React.Children.count(children),
```

Comment:
```suggestion
    itemCount: React.Children.count(children),
```

`itemCount`(예) 같은 변수명이 contextValue를 확인하지 않아도, consumer 코드만 보고 의미를 알 수 있을 거 같아 좋아보여요~!